src/components/Toolbar.vue

```diff
@@ -1,11 +1,11 @@
 <template>
   <v-toolbar :color="color" class="header">
     <template v-if="icon" #prepend>
-      <v-btn :icon="icon" size="large" @click="emit('iconClicked')" />
```

Comment:
isn't this used for the back button ?

Reply:
Do you know in which page it's actually used?
I can add some kind of check if needed, but I could not find a place where it was used

Reply:
details page of an item (such as artist details)

Reply:
shold be good now